tests/python/relax/test_ast_printer.py

```diff
@@ -432,7 +432,7 @@ def test_call_tir():
     @R.function
     def foo(x: R.Tensor(("m", "n"), "float32")):
         m, n = T.var("int64"), T.var("int64")
-        gv0 = R.call_tir("test.op.identity", (x,), R.Tensor((m, n), dtype="float32"))
+        gv0 = R.call_dps_packed("test.op.identity", (x,), R.Tensor((m, n), dtype="float32"))
```

Comment:
Maybe we should just drop this test case if it's not going to actually use `call_tir`

Reply:
Have updated the test case to use call_tir here

Reply:
Thanks